cmd/celestia-appd/cmd/root.go

```diff
@@ -246,9 +245,6 @@ func NewAppServer(logger log.Logger, db dbm.DB, traceStore io.Writer, appOpts se
 		baseapp.SetTrace(cast.ToBool(appOpts.Get(server.FlagTrace))),
 		baseapp.SetIndexEvents(cast.ToStringSlice(appOpts.Get(server.FlagIndexEvents))),
 		baseapp.SetSnapshot(snapshotStore, snapshottypes.NewSnapshotOptions(cast.ToUint64(appOpts.Get(server.FlagStateSyncSnapshotInterval)), cast.ToUint32(appOpts.Get(server.FlagStateSyncSnapshotKeepRecent)))),
-		func(b *baseapp.BaseApp) {
-			b.SetAppVersion(sdk.Context{}, appconsts.LatestVersion)
-		},
```

Comment:
@cmwaters I'd appreciate your review here. These lines are missing from the main branch so opted to delete them here which resolved some test failures. Was that correct? I don't understand the flow of setting app version before / after the cosmos SDK bump

Reply:
So in v1, the app version is hardcoded through this exact mechanism (i.e. it's manually set in the constructor). In v2 and onwards, it is read from the genesis file. It could be possible to make the same modifications to celestia-core for v1 that would allow v1.x to also read and set the app version from genesis. However, it's probably easier if you simply keep these lines around which will hardcode the version to 1

Reply:
It's not actually possible to keep these lines here because v1.x panics on this line. On v1.x, `app.ParamStore` is not defined before hitting this line.